NetworkingProject/DetailModel/Presenter/DetailPresenter.swift

```diff
@@ -7,25 +7,25 @@
 
 import Foundation
 
-protocol DetailViewProtocol: AnyObject {
-    var id: Int { get set }
-    func reloadData()
+protocol DetailMovieViewProtocol: AnyObject {
+//    var id: Int { get set }
```

Comment:
```suggestion
//    var id: Int { get set }
```
закоменченый удаляем